src/components/treePopup/index.tsx

```diff
@@ -122,33 +128,40 @@ const TreePopup: React.FC<TreePopupProps> = ({ popRef, treeInfo }) => {
       {isVisible && (
         <PopupAnchor>
           <PopupBubble>
-            {treeInfo.id !== NO_TREE_PRESENT ? (
+            <>
               <>
-                <>
-                  <TreeTitle>
-                    {isEmptyString(treeInfo.commonName)
+                <TreeTitle>
+                  {treeInfo.id !== NO_TREE_PRESENT
+                    ? isEmptyString(treeInfo.commonName)
                       ? 'Unknown Species'
-                      : treeInfo.commonName}
-                  </TreeTitle>
-                  <CloseIcon onClick={hidePopup} />
-                </>
-                <Line />
-                {!isEmptyString(treeInfo.address) && (
-                  <GreyText strong>Nearby Address</GreyText>
-                )}
-                <GreyText>{treeInfo.address}</GreyText>
+                      : treeInfo.commonName
+                    : 'Open Planting Site'}
+                </TreeTitle>
+                <CloseIcon onClick={hidePopup} />
+              </>
+              <Line />
+              {!isEmptyString(treeInfo.address) && (
+                <GreyText strong>Nearby Address</GreyText>
+              )}
+              <GreyText>{treeInfo.address}</GreyText>
+              {treeInfo.id !== NO_TREE_PRESENT ? (
                 <GreenLinkButton
                   to={`${ParameterizedRouteBases.TREE}${treeInfo.id}`}
                 >
                   More Info
                 </GreenLinkButton>
-              </>
-            ) : (
-              <>
-                <TreeTitle>Open Planting Site</TreeTitle>
-                <CloseIcon onClick={hidePopup} />
-              </>
-            )}
+              ) : (
+                <PlantRequest>
+                  Want to plant a tree here?{' '}
+                  <Link
+                    href="https://www.cityofboston.gov/311/"
+                    target="_blank"
+                  >
+                    Submit a request to the city!
+                  </Link>
+                </PlantRequest>
+              )}
```

Comment:
⛏️ Switch case might be easier to read here

Reply:
When I try this, I get an error because `switch` is a statement and it expects an expression. Should I do something like this instead
```
{(() => {
  switch(...) {}
})()}
```
or should I just pull the `switch` out of the `return` statement? Or am I doing something wrong?

Reply:
Yeah the lambda-esque syntax for the switch statement seems to be what's used in the code currently.  Aside from that it looks good!

Reply:
Okay thank you!